api/lcp/server.py

```diff
@@ -0,0 +1,351 @@
+import json
+import os
+import urlparse
+
+import requests
+from flask_babel import lazy_gettext as _
+from requests.auth import HTTPBasicAuth
+
+from api.lcp import utils
+from api.lcp.encrypt import LCPEncryptionResult, LCPEncryptorResultJSONEncoder
+from api.lcp.hash import HashingAlgorithm
+from core.model.configuration import ConfigurationBucket, ConfigurationMetadata, ConfigurationAttributeType
+
+
+class LCPServerConfiguration(ConfigurationBucket):
+    """Contains LCP License Server's settings"""
+
+    DEFAULT_PAGE_SIZE = 100
+    DEFAULT_PASSPHRASE_HINT = 'If you do not remember your passphrase, please contact your administrator'
+    DEFAULT_ENCRYPTION_ALGORITHM = HashingAlgorithm.SHA256.value
+    DEFAULT_MAX_PRINTABLE_PAGES = 10
+    DEFAULT_MAX_COPIABLE_PAGES = 2048
```

Comment:
The defaults given in https://readium.org/lcp-specs/releases/lcp/latest.html#36-identifying-rights-and-restrictions-the-rights-object are for both of these values to be unlimited; those should be the defaults here as well.

Reply:
Changed the defaults to `None`